### DISCUSSION TITLE
Conditional questions based on selected answers

### DISCUSSION THREAD

**Discussion by coxlr** [ditdot-dev/vue-flow-form#169] [Ideas] [answered]
Hi, 
Thanks for all the great work on this package it is an amazing project.
I have been looking through the source code and I don't believe this is currently an option, but if it ism my apologies. Is it possible to only show a question based on a previous answer option? 
I can see that you can jump to a question if a certain answer option is select and therefore avoiding pages and questions, but it could be a great addition if you were able to only show a specific question if a previous answer was selected as this could then be use to show a question later in the question flow.
Thanks!
Lee

**Comment by spinn** [comment, marked as answer]
Hi Lee,

the new version that we're planning to push to npm within a few days will have that capability, but only if you define questions as components, you can see an example here:

https://github.com/ditdot-dev/vue-flow-form/issues/136#issuecomment-776082649

This means that you can add v-if="..." to any question to make it conditionally appear.
